community/gnupg2/build

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh -e
+
+# fake
+export MAKEINFO=true
+touch fig2dev convert ; chmod +x fig2dev convert
+export PATH=$PWD:$PATH
```

Comment:
Why is this needed? I'll look into whether or not we can fix this a different way.

Reply:
I'm trying to avoid forcing the user to compile texinfo and friends. If there's another way than how I'm doing it of course I'd be game.

Reply:
None of the other distributions I've looked at list texinfo as a dependency. Does it build without this workaround?

Reply:
it did not build without the workaround. can't remember if it was texinfo or something other. I will try the new flags below and report back. Basically trying to avoid compiling gigabytes of extra crud, though.

Reply:
I don't know if it's just my KISS installation or what but gnupg2 in my personal repo doesn't have `MAKEINFO=true` and it built successfully. Do you have any `texinfo`-related packages in your system?